src/resources/js/commerce/gateway/toggler.js

```diff
@@ -155,6 +155,11 @@ tribe.tickets.commerce.gateway.toggler = tribe.tickets.commerce.gateway.toggler
 	 * @since TBD
 	 */
 	obj.showAdditional = () => {
+		// If additional options already open, let's close it and show default.
+		if( obj.toggles.additional.hasClass( obj.selectors.toggleOpen.className() ) ) {
```

Comment:
is `toggleOpen.className()` correct?

Reply:
Both `is()` and `hasClass()` would work in this case, but `hasClass()` performs better since it only looks at classes.